pueue/src/client/query/mod.rs

```diff
@@ -93,6 +94,24 @@ impl QueryResult {
             Rule::column_label => task1.label.cmp(&task2.label),
             Rule::column_command => task1.command.cmp(&task2.command),
             Rule::column_path => task1.path.cmp(&task2.path),
+            Rule::column_enqueue_at => {
+                fn enqueue_date(task: &Task) -> DateTime<Local> {
+                    match &task.status {
+                        TaskStatus::Queued { enqueued_at, .. }
+                        | TaskStatus::Running { enqueued_at, .. }
+                        | TaskStatus::Paused { enqueued_at, .. }
+                        | TaskStatus::Done { enqueued_at, .. }
+                        | TaskStatus::Stashed {
+                            enqueue_at: Some(enqueued_at),
+                            ..
+                        } => *enqueued_at,
+                        // considered far in the future when no explicit date:
+                        _ => DateTime::<Utc>::MAX_UTC.into(),
+                    }
+                }
+
+                enqueue_date(task1).cmp(&enqueue_date(task2))
+            }
```

Comment:
`enqueue_at` and `enqueued_at` are two different things though.

One is when it has been enqueued, the other one when it'll be enqueued.

Reply:
True, but the column that's shown in the `pueue status` column is the `enqueue_at` field. I'm a bit concerned that it might be non-intuitive for users on how those commands are sorted, since they don't see the `enqueued_at` field.

Not showing `enqueued_at` in `pueue status` output was a deliberate decision as this has never been asked for and the `pueue status` table should be as narrow as possible to make it somewhat

Though this isn't a strong feeling on my end. So if you think that this feels intuitive, I would be ok with the current implementation :)

Reply:
Fair point, the fallback behavior would be just as confusing.

You convinced me, let's keep it the way it is right now :)